doc/cookbook/source/examples/gaussian_processes/gaussian_process_regression.rst

```diff
@@ -0,0 +1,59 @@
+===========================
+Gaussian Process Regression
+===========================
+
+A Gaussian Process is the extension of the Gaussian distribution to infinite dimensions. It is specified by a mean function :math:`m(\mathbf{x})` and a covariance kernel :math:`k(\mathbf{x},\mathbf{x}')` (where :math:`\mathbf{x}\in\mathcal{X}` for some input domain :math:`\mathcal{X}`). They define a distribution over real valued functions :math:`f(\cdot)`:
+
+.. math::
+	f(\mathbf{x})\sim\mathcal{GP}(m(\mathbf{x}), k(\mathbf{x},\mathbf{x}))
+
+Any finite set of such function values :math:`\mathbf{f}` is jointly Gaussian with mean specified by the mean function evaluated at the inputs :math:`\mathbf{x}_i` and covariance matrix :math:`\mathbf{C}` with entries :math:`\mathbf{C}_{ij}=k(\mathbf{x}_i,\mathbf{x}_j)`.
+
+In our regression model, we take our target variable :math:`\mathbf{y}` to be our (latent) function values :math:`\mathbf{f}` plus Gaussian noise:
+
+.. math::
```

Comment:
I would put the prediction integral that you have below up here.
"Given training data, the label for a new point can be predicted as...."